### ISSUE TITLE
Null safety is not fully supported

### ISSUE THREAD

**Issue by jukqaz** [lschmierer/android_play_install_referrer#11]
```
static Future<ReferrerDetails?> get installReferrer async {
    final Map? details = await _channel.invokeMethod('getInstallReferrer');

    if (details == null) {
      return null;
    }

    return ReferrerDetails(
      details['installReferrer'],
      details['referrerClickTimestampSeconds'],
      details['installBeginTimestampSeconds'],
      details['referrerClickTimestampServerSeconds'],
      details['installBeginTimestampServerSeconds'],
      details['installVersion'],
      details['googlePlayInstantParam'],
    );
  }
```

if details map has null value, happen casting exception in ReferrerDetails class's property. (Null to others)

**Comment by lschmierer** [comment]
Would changing
```Dart
  late final String _installReferrer;
  late final int _referrerClickTimestampSeconds;
  late final int _installBeginTimestampSeconds;
  late final int _referrerClickTimestampServerSeconds;
  late final int _installBeginTimestampServerSeconds;
  late final String _installVersion;
  late final bool _googlePlayInstantParam;
```
to

```Dart
  late final String? _installReferrer;
  late final int? _referrerClickTimestampSeconds;
  late final int? _installBeginTimestampSeconds;
  late final int? _referrerClickTimestampServerSeconds;
  late final int? _installBeginTimestampServerSeconds;
  late final String? _installVersion;
  late final bool? _googlePlayInstantParam;
```
fix this?

**Comment by lschmierer** [comment]
@jukqaz can you check if this is fixed in release 0.1.1?

Closing this for now. Feel free to reopen if there are still issues.
